mlflow/models/evaluation/base.py

```diff
@@ -961,6 +962,17 @@ def evaluate(
         - The evaluation dataset label values must be numeric or boolean, all feature values
           must be numeric, and each feature column must only contain scalar values.
 
+     - Limitations when the model is loaded with a non-local ``env_manager``:
+        - When environment restoration is enabled, the model that is passed to the default evaluator
```

Comment:
Can we rephrase or reorder these sentences to clarify that "non-local env_manager" == "environment restoration is enabled"? Right now, they sound like different things.